### DISCUSSION TITLE
storybook/writing-stories

### DISCUSSION THREAD

**Discussion by giscus** [howdy-mj/blog-v2#50] [Announcements]
# storybook/writing-stories

스토리북에서 stories를 어떻게 작성하는지 알아보자.

해당 글은 react `^17.0.1`, react-scripts `4.0.1`, @storybook/react `^6.1.8`, styled-components `^5.2.1`, typescript `^4....

https://www.howdy-mj.me/storybook/writing-stories

**Comment by chucoding** [comment]
SB 사용시 패키지 구조를 어떻게 잡는지 몰랐는데 좋은 참고가 되었습니다 감사합니다:)
